src/shared/api/services/appletsService.ts

```diff
@@ -10,6 +10,8 @@ import {
   AppletAndActivitiesDetailsResponse,
   AppletAssignmentsRequest,
   AppletAssignmentsResponse,
+  AppletBaseInfoRequest,
+  AppletBaseInfoResponse,
```

Comment:
I don't really know how these missing imports passed the original PR. Maybe there is something wrong with the CI running type-checks? 🤔

Reply:
You're right! `tsc` is not being run on the mobile app because there's no deployment script (on the web apps, it's being run as a by-product of setting up the Amplify preview environment). Could you fix this in this PR by cherry-picking [this commit](https://github.com/ChildMindInstitute/mindlogger-app-refactor/pull/886/commits/d55ddc76c06204568736e45c3b3891cea2f864d0) into your PR?